src/qforte/adapters/molecule_adapters.py

```diff
@@ -125,18 +125,25 @@ def create_psi_mol(**kwargs):
     frozen_core = p4_wfn.frzcpi().sum()
     frozen_virtual = p4_wfn.frzvpi().sum()
 
-    # Get symmetry information
+    # Get orbital symmetry information and construct hf reference
     orbitals = []
     for irrep, block in enumerate(p4_wfn.epsilon_a_subset("MO", "ACTIVE").nph):
         for orbital in block:
             orbitals.append([orbital, irrep])
 
     orbitals.sort()
+    occ_alpha_per_irrep = p4_wfn.occupation_a().nph
+    occ_beta_per_irrep = p4_wfn.occupation_b().nph
+    count_per_irrep = list(p4_wfn.frzcpi().to_tuple())
```

Comment:
This is a nitpick, but why is this cast to a list needed? `count_per_irrep` doesn't need to be mutated, so I don't see what advantages `list` offers over the native `tuple`.

Reply:
Actually, `count_per_irrep' is mutated on line 144, hence the casting to a list.